app/views/common/_google_universal_analytics.html.erb

```diff
@@ -4,4 +4,6 @@
   auto_link_domains: SiteSetting.ga_universal_auto_link_domains
 } %>
 
+<script async src="https://www.googletagmanager.com/gtag/js?id=<%= SiteSetting.ga_universal_tracking_code %>"></script>
```

Comment:
Are including the gtag script even if the site setting is empty?

Reply:
No this partial gets loaded only if the site setting is present.